etc/adminhtml/system.xml

```diff
@@ -114,7 +114,7 @@
                     <comment>Enable customers to pay for products in Tesco Lotus Stores.</comment>
                     <field id="active" translate="label comment" type="select" sortOrder="200" showInDefault="1" showInWebsite="1" showInStore="1">
                         <label>Enable/Disable</label>
-                        <comment>Enable Tesco Bill Payment Solution.</comment>
+                        <comment>Enable Tesco Lotus Bill Payment method.</comment>
```

Comment:
@danfowler just wonder if we switch to use `Tesco Lotus Bill Payment` instead of `Tesco Bill Payment`? (if so then I'll update on WC plugin too)

Reply:
"Tesco Lotus" seems closer to the mark than "Tesco" as (a) that's the [name](https://en.wikipedia.org/wiki/Tesco_Lotus) of the company and (b) it's in the source type `bill_payment_tesco_lotus`.